tool/tctl/common/resource_command.go

```diff
@@ -376,6 +389,34 @@ func (rc *ResourceCommand) createUser(client auth.ClientI, raw services.UnknownR
 	return nil
 }
 
+// createClusterConfig implements `tctl create -f cc.yaml` command,
+// overwriting the current cluster configuration.
+func (rc *ResourceCommand) createClusterConfig(client auth.ClientI, raw services.UnknownResource) error {
+	if !rc.force {
+		return trace.AlreadyExists("cluster configuration can be only overwritten or updated")
+	}
+
+	existingConfig, err := client.GetClusterConfig()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	newConfig, err := services.GetClusterConfigMarshaler().Unmarshal(raw.Raw)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	if newConfig.GetName() != existingConfig.GetName() {
+		return trace.CompareFailed("cluster config resource must have the name %q", existingConfig.GetName())
```

Comment:
error messages should be consistent with the resource type (`cluster configuration`)